### PR TITLE
BZ2018063: QEMU guest agent - RN (all versions)

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=2018063

Add RN and QEMU guest agent known issue to RN

Preview: https://deploy-preview-227--forklift-documentation.netlify.app/documentation/doc-release_notes/master/#known-issues-22_forklift

### CODE REVIEW

**Comment by apinnick** [comment]
@istein1 Ready for QE. Please review.

File: documentation/modules/rn-2.2.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+// .TBD
+//
+// TBD
+
+[id="known-issues-22_{context}"]
+== Known issues
+
+.QEMU guest agent is not installed on migrated VMs
+
+The QEMU guest agent is not installed on migrated {rhv-full} VMs. Workaround: Install the QEMU guest agent with a post-migration hook. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2018062[*BZ#2018062*])
```
Comment (istein1) [inline documentation/modules/rn-2.2.adoc:29]: QEMU guest agent installation is missing for both RHV & VMware.

**Comment by istein1** [comment]
Will this update appear for all MTV past versions too (MTV-2.0, 2.1)?

File: documentation/modules/rn-2.2.adoc (hunk 1 above)
Comment (apinnick) [reply documentation/modules/rn-2.2.adoc:29]: OK. I'll remove the RHV reference.

**Comment by apinnick** [comment]
> Will this update appear for all MTV past versions too (MTV-2.0, 2.1)?

@istein1 Yes. I can copy the change.

**Pull request merged**